assets/js/buttons-examples.js

```diff
@@ -199,6 +199,12 @@ $(document).ready(function () {
           classes: "scroll-to-top-btn",
           text: "",
         },
+        {
+          type: "star",
+          label: "Star Button",
+          classes: "star-btn",
```

Comment:
set classes to `""` and add a variation object (you can check social buttons to see how it's done)

Reply:
I added a variation and changed the active color to yellow. But I still have an issue, is it possible to get rid of the rounded and block examples?
![image](https://user-images.githubusercontent.com/16581093/94937552-19de1880-04a6-11eb-9d13-b8d4bf7852e8.png)

Reply:
I need to see what you did for me to let you know. Can you commit your changes?